### ISSUE TITLE
Loading local mp3 on iPhone doesn't work

### ISSUE THREAD

**Issue by evelyncplum** [Mr-Pepe/tingting#4]
Loading local mp3 on iPhone doesn't work

**Comment by Mr-Pepe** [comment]
Can you run the app in debug mode and copy the error message here that appears when you try to load the file?

**Comment by evelyncplum** [comment]
There is no error message?

**Comment by Mr-Pepe** [comment]
So when you run the app with "flutter run" (no release flag), it doesn't print anything to the terminal?

**Comment by evelyncplum** [comment]
Nope.

**Comment by Mr-Pepe** [comment]
Okay, can you explain in more detail what happens? 
Can you press the button to select an audio file? 
Can you select an audio file?
Do the buttons to control audio show up?
Where does it fail?

**Comment by evelyncplum** [comment]
I created the file using the "Voice Memos" app, which is a default Apple app. I had to export that from the app to my "Files" app where I could choose it from TingTing. 

Yes, I can press the button to select the audio file, and I can select the file but after this, when it returns to the app, nothing happens. There are no buttons to control the audio.

**Comment by Mr-Pepe** [comment]
When I run "flutter run", I get this kind of output:

![image](https://user-images.githubusercontent.com/26527091/89607965-5de9de00-d8a6-11ea-8e5c-1c2488e409ef.png)

Does your terminal just stay blank and not show any output while the app is running on your phone?

There are apparently [issues](https://github.com/flutter/flutter/issues/41133) with logging on physical iOS devices, so it might be related to that.

**Comment by Mr-Pepe** [comment]
While looking at another issue I found that file names including question marks or hashtags can not be loaded. Have you tried different files? Can you not load any?

**Comment by evelyncplum** [comment]
Yes, my terminal stays blank, nothing happens. Until I disconnect, obviously, and then it says connection is lost.

Yes, I tried another file that I got from the internet and it also did not work.

**Comment by Mr-Pepe** [comment]
In that case it takes a bit more effort for you to track down the problem. Let me know if you are interested. Otherwise I'll need to find someone with a MacBook and an iPhone who let's me test it, but that might take a while.

In the meantime you can try and see if the audio generation from text works on iOS, if you are interested.

**Comment by Mr-Pepe** [comment]
Does it print something if you start the app without the release flag? 

Edit: Nvm I said that earlier already

**Comment by evelyncplum** [comment]
<img width="800" alt="Screen Shot 2020-08-16 at 4 12 41 PM" src="https://user-images.githubusercontent.com/62696685/90344000-5a0f3780-dfdb-11ea-9909-e2b083343b20.png">

Here's what happens when I try to load an audio file that I downloaded from the internet.

**Comment by evelyncplum** [comment]
<img width="801" alt="Screen Shot 2020-08-16 at 4 16 58 PM" src="https://user-images.githubusercontent.com/62696685/90344079-eae61300-dfdb-11ea-8ba2-61e9c2a0ce43.png">

Slightly different error when I tried loading an audio file that I'd made myself locally with my phone.

**Comment by Mr-Pepe** [comment]
There is a fix in the branch `Mr-Pepe/#4-Loading-local-mp3-on-iPhone-doesn't-work`. Could you test that? 
You have to switch to the branch first by running `git checkout Mr-Pepe/#4-Loading-local-mp3-on-iPhone-doesn't-work`
